tests/integration/testlib/ovslib.py

```diff
@@ -91,6 +91,16 @@ def _get_ports(self):
             OVSBridge.PORT_SUBTREE, []
         )
 
+    def set_port_option(self, new_option):
+        for port_option in self._bridge_iface[OVSBridge.CONFIG_SUBTREE].get(
+            OVSBridge.PORT_SUBTREE, []
+        ):
+            if (
+                port_option[OVSBridge.Port.NAME]
+                == new_option[OVSBridge.Port.NAME]
```

Comment:
Can you make this an explicit argument so it will be clear and explicit that this is the port to change?